tests/main/postrm-purge/task.yaml

```diff
@@ -4,19 +4,28 @@ systems: [-ubuntu-core-16-*]
 
 execute: |
     echo "When some snaps are installed"
-    . $TESTSLIB/snaps.sh
+    . "$TESTSLIB/snaps.sh"
```

Comment:
right, but as you say it's good practice, and shellcheck will complain about these, so why not do it?